### DISCUSSION TITLE
AI Summary

### DISCUSSION THREAD

**Discussion by arckdash** [dailydotdev/daily#1258] [Feature Requests]
With the hype of AI & LLMs, it'd be great if you guys could add a feature that lets you get a summary of posts / bookmarks, why? Well it's quite easy to pile them all up and then forget you even bookmarked them...

Whereas bookmarking makes it great to save something to read later, when it's been too long, it'd be pretty awesome to leverage the power of AI to get a quick summary and determine if something is still relevant to you!

Also would be nice to make it do search, like:
find the post about react native internal storage

finding the needle in the haystack is always a PITA xD

Thanks for the awesome work! :)

**Comment by nimrodkra** [comment]
We have a TLDR feature on each post to help users quickly understand the main points, which should assist in summarizing your bookmarked posts. Does it solve your problem or is it something else you're looking for?

![Screenshot 2024-07-07 at 9 45 07](https://github.com/dailydotdev/daily/assets/41470823/75259c41-e1ed-4531-b869-4853997db88f)
